profile/rwahs.xml

```diff
@@ -7806,9 +7806,11 @@
       <settings>
         <setting name="show_empty_values"><![CDATA[1]]></setting>
       </settings>
-      <userAccess>
-        <permission user="administrator" access="edit"/>
```

Comment:
This display was created through the UI and exported. Use specific permissions are dody.